app/models/puppet_error.rb

```diff
@@ -0,0 +1,7 @@
+class PuppetError < MailNotification
```

Comment:
Very understandable, thanks! 

nitpick - Can you move it to `/app/models/mail_notifications/puppet_error.rb`?